### ISSUE TITLE
Устаревшая версия YandexAds SDK

### ISSUE THREAD

**Issue by maningame** [maximnara/cordova-plugin-yandex-ads#12]
Сейчас в проекте используется 6.2.0, а доступна уже 6.4.1

**Comment by maximnara** [comment]
Среди минорных изменений нет ломающих?

**Comment by maningame** [comment]
Нет, одни лишь [повышения стабильности](https://ads.yandex.com/helpcenter/ru/dev/android/changelog-android). У себя собрал билд с 6.4.1 пока полёт нормальный.

**Comment by maximnara** [comment]
Спасибо, поменяю
